car.capnp

```diff
@@ -458,6 +458,7 @@ struct CarParams {
   struct SafetyConfig {
     safetyModel @0 :SafetyModel;
     safetyParam @1 :Int16;
+    unsafeMode @2 :Int16;
```

Comment:
shouldn't go in this struct as it doesn't vary per-panda

Reply:
Ah I see. It should be added to pandaState as well so we can verify what's set on the panda side as well, right?

Reply:
On the panda side, we set each panda to have the unsafe_mode. So are we assuming that the function (`set_unsafe_mode` in panda.cc) will never fail? If it does, how do we know which panda to compare its unsafe_mode with the single one in carParams?